### PR TITLE
Zy0n/subsquid sepolia

### PR DESCRIPTION
**Pull request by zy0n**
This includes Sepolia enabled for PPOI's use of railgun-txids.
All tests passing

### CODE REVIEW

File: src/services/railgun/railgun-txids/__tests__/railgun-txid-sync-graph-v2.test.ts [hunk 1]
```diff
@@ -18,6 +18,11 @@ describe('railgun-txid-sync-graph', () => {
       launchBlock: 1000,
       launchTimestamp: 1000,
     };
+
+    NETWORK_CONFIG[NetworkName.EthereumSepolia].poi = {
```
Comment (jacobmakarsky) [inline src/services/railgun/railgun-txids/__tests__/railgun-txid-sync-graph-v2.test.ts:22]: I think this should match shared-models: https://github.com/Railgun-Community/shared-models/blob/3baba0e95f2dea5ff3e61f82bbdadc4bd8cc49b9/src/models/network-config.ts#L502

File: src/services/railgun/railgun-txids/__tests__/railgun-txid-sync-graph-v2.test.ts [hunk 2]
```diff
@@ -65,4 +70,50 @@ describe('railgun-txid-sync-graph', () => {
       '065bcb1a9d4cfa110f05b480f79f27fe2ad672868d3d1bdec05df2ddaec8333d',
     ]);
   }).timeout(20000);
-  });
+
+  it('Should pull railgun txs subgraph query - Sepolia', async () => {
```
Comment (jacobmakarsky) [inline src/services/railgun/railgun-txids/__tests__/railgun-txid-sync-graph-v2.test.ts:74]: subsquid

File: src/services/transactions/__tests__/tx-cross-contract-calls.test.ts [hunk 3]
```diff
@@ -31,6 +31,7 @@ import {
   MOCK_DB_ENCRYPTION_KEY,
   MOCK_ERC20_RECIPIENTS,
   MOCK_ETH_WALLET_ADDRESS,
+  MOCK_FALLBACK_PROVIDER_JSON_CONFIG_POLYGON,
```
Comment (jacobmakarsky) [inline src/services/transactions/__tests__/tx-cross-contract-calls.test.ts:34]: Should this use Sepolia as well

File: src/services/transactions/__tests__/tx-cross-contract-calls.test.ts [hunk 4]
```diff
@@ -148,7 +149,8 @@ describe('tx-cross-contract-calls', () => {
   before(async function run() {
     this.timeout(60_000);
     await initTestEngine();
-    await initTestEngineNetworks();
+    await initTestEngineNetworks(NetworkName.Polygon, MOCK_FALLBACK_PROVIDER_JSON_CONFIG_POLYGON);
```
Comment (jacobmakarsky) [inline src/services/transactions/__tests__/tx-cross-contract-calls.test.ts:152]: Also sepolia possibly

File: src/services/transactions/__tests__/tx-cross-contract-calls.test.ts (hunk 3 above)
Comment (zy0n) [reply src/services/transactions/__tests__/tx-cross-contract-calls.test.ts:34]: No, this is to fix failing tests. The sepolia test accounts doesn't have whats required in these tests. They are fine to run on polygon.

There is no need to 're-write' these tests with new sepolia data, the current tests are valid and viable.

File: src/services/transactions/__tests__/tx-cross-contract-calls.test.ts (hunk 4 above)
Comment (zy0n) [reply src/services/transactions/__tests__/tx-cross-contract-calls.test.ts:152]: [please see](https://github.com/Railgun-Community/wallet/pull/75#discussion_r1617233461)

File: src/services/railgun/railgun-txids/__tests__/railgun-txid-sync-graph-v2.test.ts (hunk 2 above)
Comment (zy0n) [reply src/services/railgun/railgun-txids/__tests__/railgun-txid-sync-graph-v2.test.ts:74]: Okay.

**Pull request merged**